lib/ecto/query.ex

```diff
@@ -529,6 +529,15 @@ defmodule Ecto.Query do
     end
   end
 
+  @doc """
+  Returns `true` if `query` has named binding `name`; otherwise returns `false`.
+
+  For more information on named bindings see "Named bindings" in this module doc.
```

Comment:
Let's say it is the same as `has_named_binding?` but allowed in guards. I also don't think we need `@doc guard: true` (Elixir uses it for docs but it is not required).

Reply:
done!